noir-projects/noir-contracts/contracts/inclusion_proofs_contract/src/main.nr

```diff
@@ -65,15 +65,15 @@ contract InclusionProofs {
         // docs:end:get_note_from_pxe
 
         // 2) Prove the note inclusion
-        if (use_block_number) {
+        let header = if (use_block_number) {
             // docs:start:prove_note_inclusion
```

Comment:
This doc string should probably be moved down? 👀 Otherwise just showing the get header when wanting to show note inclusion.